gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsInstrumentation.java

```diff
@@ -569,7 +573,6 @@ public void seekForwards(long skipped) {
       if (skipped > 0) {
         bytesSkippedOnSeek.addAndGet(skipped);
       }
-      seekOperations.incrementAndGet();
```

Comment:
Why we are removing this? Similar comment fir `seekBackwards` too.

nit: update the java doc of this function.